DigraphyApi/Repository/OrderRepository.cs

```diff
@@ -0,0 +1,19 @@
+using DigraphyApi.Data;
+using DigraphyApi.Interfaces;
+using DigraphyApi.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace DigraphyApi.Repository;
+
+public class OrderRepository(AppDbContext context) : IOrderRepository
+{
+    public async Task<ICollection<Order>> GetOrdersAsync(bool verified)
+    {
+        return await context.Orders.Where(o => !o.IsVerified && verified == false).OrderBy(o => o.Id).ToListAsync();
+    }
+
+    public async Task<ICollection<Order>> GetOrdersByCollectionIdAsync( int collectionId, bool verified)
+    {
+        return await context.Orders.Where(o => !o.IsVerified && verified == false && o.Collection.Id  == collectionId).OrderBy(o => o.Id).ToListAsync();
```

Comment:
I don't think these boolean expression does what you want? If `verified` is true this will never return anything as the `verified == false` condition will be `false` and because it's ANDed the whole condition becomes false.

What I think we want is for verified to be nullable. If it's null then we're not filtering by `verified` and can therefore exclude that part of the query (See my previous comment). Otherwise, we simply want to check that `o.IsVerified == verified`.